frontend/src/components/Error.tsx

```diff
@@ -1,24 +1,16 @@
-// This Component requires some changes and styling
-import Alert from "@mui/material/Alert";
-import AlertTitle from "@mui/material/AlertTitle";
-import Stack from "@mui/material/Stack";
-import { ERROR_TITLE, PRODUCTS_NOT_FOUND_MESSAGE } from "./constants/text";
-
-const ProductsNotFoundAlert = () => {
-
-    const alertContainerStyles = {
-        width: "100%",
-        marginTop: "200px",
-    };
+// This component needs styling 
+// Material UI 
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import Stack from '@mui/material/Stack';
 
+export default function DescriptionAlerts() {
     return (
-        <Stack sx={ alertContainerStyles } spacing={ 2 }>
+        <Stack style={ { marginTop: "200px" } } sx={ { width: '100%' } } spacing={ 2 }>
             <Alert severity="error">
-                <AlertTitle>{ ERROR_TITLE }</AlertTitle>
-                { PRODUCTS_NOT_FOUND_MESSAGE }
+                <AlertTitle>Error</AlertTitle>
+                Product was not found— <strong>check it out!</strong>
```

Comment:
let's keep this in the constants file

Reply:
I fixed this in another PR,  have not finished working on it yet.